en/config/actuators.md

```diff
@@ -140,32 +140,60 @@ Note that you will need to also ensure that the ESC associated with bidirectiona
 The control surfaces section of the geometry panel lets you set the number and types of control surfaces that are present on the vehicle.
 You may also need to set trim and slew rate values in some cases.
 More advanced users can also configure the roll scale, yaw scale, and pitch scale (generally the defaults are acceptable, and this is not needed).
-
 An "example" control surface section for a vehicle with two ailerons is shown below.
 Note that ailerons only affect roll, so the pitch and yaw fields are disabled.
 
 ![Control Surface Setup Example](../../assets/config/actuators/control_surfaces_geometry.png)
 
+:::note
+Only the most common settings are displayed by default.
+Select the **Advanced** checkbox in the top right corner of the view to display all settings.
+:::
+
 The fields are:
 
 - `Control Surfaces`: The number of control surfaces (set this first!)
 - `Type`: The type of each control surface: `LeftAileron`, `RightAileron`, `Elevator`, `Rudder`, `Left Elevon`, `Right Elevon`, `Left V-Tail`, `Right V-Tail`, `Left Flap`, `Right Flap`, `Airbrakes`, `Custom`.
-- `Roll scale`: Effectiveness of actuator around roll axis (normalised: -1 to 1).
+- `Roll Torque`: Effectiveness of actuator around roll axis (normalised: -1 to 1).
   [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
-- `Pitch scale`: Effectiveness of actuator around pitch axis (normalised: -1 to 1).
+- `Pitch Torque`: Effectiveness of actuator around pitch axis (normalised: -1 to 1).
   [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
-- `Yaw scale`: Effectiveness of actuator around yaw axis (normalised: -1 to 1).
+- `Yaw Torque`: Effectiveness of actuator around yaw axis (normalised: -1 to 1).
   [Generally you should use the default actuator value](#actuator-roll-pitch-and-yaw-scaling).
 - `Trim`: An offset added to the actuator so that it is centered without input.
   This might be determined by trial and error.
-- `Slew Rate`: Minimum time allowed for the motor/servo signal to pass through the full output range, in seconds.
+- (Advanced) `Slew Rate`: Minimum time allowed for the motor/servo signal to pass through the full output range, in seconds.
   - The setting limits the rate of change of an actuator (if not specified then no rate limit is applied).
      It is intended for actuators that may be damaged if they move too fast — such as the tilting actuators on a tiltrotor VTOL vehicle.
   - For example, a setting of 2.0 means that the motor/servo will not be commanded to move from 0 to 1 at a rate that completes the operation in less than 2 seconds (in case of reversible motors, the range is -1 to 1).
+- (Advanced) `Flap Scale`: How much this actuator is deflected at the "full flaps configuration" [0, 1] (see [Flaps and Spoilers Configuration](#flaps-and-spoilers-configuration) below).
```

Comment:
Is this in PX4 v1.14? I can't get them to display on whatever I have on my hardware (that won't flash), though I was able to get them in simulator.